### ISSUE TITLE
sidebar directive in the new doc theme not producing a sidebar

### ISSUE THREAD

**Issue by hoechenberger** [mne-tools/mne-python#9031]
It appears `.. sidebar::` will not actually create a sidebar anymore – which **does** make sense IMHO with the new layout; however maybe we should replace "sidebars" with "info boxes" now?

Source:
<img width="500" alt="Screen Shot 2021-03-12 at 12 24 06" src="https://user-images.githubusercontent.com/2046265/110934250-0f199080-832e-11eb-9e97-a7a30bd0f4a8.png">

Rendering:
<img width="500" alt="Screen Shot 2021-03-12 at 12 23 42" src="https://user-images.githubusercontent.com/2046265/110934271-15a80800-832e-11eb-8b69-87ded7515050.png">

**Comment by larsoner** [comment]
Can you look to see what needs to be changed in https://github.com/larsoner/sphinx-bootstrap-divs which we vendor in https://github.com/mne-tools/mne-python/tree/main/doc/sphinxext/sphinx_bootstrap_divs ?

**Comment by larsoner** [comment]
... actually we shouldn't even really use that anymore since it's for sphinx-bootstrap-theme, which uses BS3, and pydata-sphinx-theme uses BS4 I think.

I'd propose adding whatever the correct thing is (if it is info boxes) to the upstream pydata-sphinx-theme to see if they'd want it. If they do then let's implement it there, if they don't then we can vendor in our own `doc/sphinxext/*` somewhere.

**Comment by drammock** [comment]
> actually we shouldn't even really use that anymore since it's for sphinx-bootstrap-theme, which uses BS3, and pydata-sphinx-theme uses BS4 I think.

FYI I updated `doc/sphinxext/sphinx_bootstrap_divs` when we switched themes, to be BS4 compliant.  (not certain I did it fully or in the best way, but I at least made it work).

The real problem here was that the sidebars used to be put in a `div` and now they're in an `aside` tag instead, and our CSS selector was too specific.  I've pushed a fix as part of #9036.
